Makefile

```diff
@@ -43,6 +43,7 @@ deploy: manifests
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./..." output:crd:artifacts:config=deploy/crds/
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./..." output:crd:artifacts:config=chart/templates/crds/
```

Comment:
I have added this too make sure, both helm folder crd and deploy folder crds are updated at once.